dao/src/main/java/greencity/entity/TextTranslation.java

```diff
@@ -0,0 +1,26 @@
+package greencity.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "text_translations")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class TextTranslation {
```

Comment:
`TextTranslation` and `TitleTranslation` could extend some abstract class `Translation` as they have common fields. It would be interesting to map them to they tables. Please consider as separate ticket.